### PR TITLE
create farm selection page and nav routes

### PR DESCRIPTION
**Pull request by SathursanS**
This PR creates the basic Farm Selection page. 
Rebased with @PreyanshK  PR and connects flow together.

https://github.com/adshayanB/ECE452/assets/36520014/a03fed30-f5ac-447c-ac45-0fa4a5c6957d

### CODE REVIEW

File: app/src/main/java/com/example/farmeraid/farm_selection/model/FarmSelectionModel.kt
Comment (PreyanshK) [inline app/src/main/java/com/example/farmeraid/farm_selection/model/FarmSelectionModel.kt:1]: Do we need this model? Seems like it's an empty class

File: app/src/main/java/com/example/farmeraid/farm_selection/model/FarmSelectionModel.kt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package com.example.farmeraid.farm_selection.model
+
+class FarmSelectionModel {
```
Comment (PreyanshK) [inline app/src/main/java/com/example/farmeraid/farm_selection/model/FarmSelectionModel.kt:3]: Do we need this model? It looks like an empty class.

**Review by PreyanshK — APPROVED** [review]
Other than the empty model file, everything else looks good to me

**Review by RishanR — APPROVED** [review]
Agree with Preyansh about deleting the empty class. Otherwise, its looking solid!

**Pull request merged**